docs/reference/commandline/build.md

```diff
@@ -277,7 +277,7 @@ flag](../run.md#specifying-custom-cgroups).
 
 Using the `--ulimit` option with `docker build` will cause each build step's
 container to be started using those [`--ulimit`
-flag values](../run.md#setting-ulimits-in-a-container).
+flag values](./run.md#set-ulimits-in-container-ulimit).
```

Comment:
Hi @vdemeester  we can not write #set-ulimits-in-container---ulimit, because when use make docs, and check that, it not work